docs/backend-requests/versioning/overview.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Versioning Overview
+description: A brief guide on how versioning works in Clerk.
+---
+
+# Versioning
+
+## Introduction
+
+When backwards incompatible changes are necessary for an API, new versions are released to prevent disruption to existing applications. These versions are identified by their release date, such as 2024-01-02. Both the frontend and backend APIs follow the same version.
```

Comment:
🙃 Maybe use the initial version as an example instead?

```suggestion
When backwards incompatible changes are necessary for an API, new versions are released to prevent disruption to existing applications. These versions are identified by their release date, such as 2021-02-05. Both the frontend and backend APIs follow the same version.
```

Reply:
Good call, fixed 🙏